jax/experimental/global_device_array.py

```diff
@@ -402,6 +435,33 @@ def from_batched_callback_with_devices(
       mesh_axes: MeshAxes,
       data_callback: Callable[[Sequence[Tuple[Index, Tuple[Device, ...]]]],
                               Sequence[DeviceArray]]):
+    """Constructs a GlobalDeviceArray via batched DeviceArrays fetched from ``data_callback``.
+
+    Like ``from_batched_callback``, except the callback function is responsible for returning on-device data (e.g. by calling ``jax.device_put``).
+
+    Example::
```

Comment:
`Examples:`

Reply:
No, it doesn't become a code block if there is no double colon.